data/projects.js

```diff
@@ -306,5 +306,26 @@ coala should support generating metrics for your code.",
 		"tags" : ["Web", "CI"],
 		"markdown": "gitmate_gitlab.md",
 		"collaborating_projects" : ["coala", "GitMate", "IGitt"]
+	},
+	{
+		"name" : "Improve installation of coala and coala-bears",
+		"desc" : "This project is about improving the \
+		[bear installation tool](https://gitlab.com/coala/cib) and \
+		working around platform dependent requirements.",
+		"requirements" : [
+				"The participant should have one patch to [coala-bears](https://github.com/coala/coala-bears) accepted.",
+				"Huge plus is merging [this issue](https://github.com/coala/coala-bears/issues/845)",
```

Comment:
I'd remove that, sends the message that only that one student who is merging that issue has a good chance of getting this which isn't true